awebox/mdl/aero/induction_dir/vortex_dir/tools.py

```diff
@@ -31,329 +31,144 @@
+    return wg_local
 
-    horz_filaments = filaments.T
+def evaluate_symbolic_on_segments_and_sum(filament_fun, segment_list):
 
-    return horz_filaments
+    n_filaments = segment_list.shape[1]
+    filament_map = filament_fun.map(n_filaments, 'openmp')
+    all = filament_map(segment_list)
 
+    total = cas.sum2(all)
 
-def get_filament_list(options, wind, variables, architecture):
-    n_k = options['aero']['vortex']['n_k']
-    d = options['aero']['vortex']['d']
-    periods_tracked = options['aero']['vortex']['periods_tracked']
-    u_vec_ref = wind.get_velocity_ref() * vect_op.xhat()
+    return total
 
-    filament_list = get_list_of_all_filaments(variables['xd'], variables['xl'], architecture, u_vec_ref,
-                                                           periods_tracked, n_k, d)
+def get_position_scale(options):
 
-    return filament_list
+    wx_scale = options['induction']['vortex_position_scale']
+    return wx_scale
 
+def get_velocity_scale(options):
+    dwx_scale = options['induction']['vortex_u_ref']
+    return dwx_scale
 
-def get_list_of_filaments_by_kite_and_ring(options, variables, wind, kite, parent, rdx):
-    variables_xd = variables['xd']
-    variables_xl = variables['xl']
+def get_strength_scale(options):
+    wg_scale = options['induction']['vortex_gamma_scale']
+    return wg_scale
 
-    n_k = options['aero']['vortex']['n_k']
-    d = options['aero']['vortex']['d']
-    periods_tracked = options['aero']['vortex']['periods_tracked']
-    u_vec_ref = wind.get_velocity_ref() * vect_op.xhat()
+def append_bounds(g_bounds, fix):
 
-    infinite_time = 1000.
+    if (type(fix) == type([])) and fix == []:
+        return g_bounds
 
-    padded_strengths = get_padded_strengths_by_kite(variables_xl, n_k, d, periods_tracked, kite, parent)
-    padded_points_int = get_padded_points_by_kite_and_tip(variables_xd, n_k, d, periods_tracked, kite, parent,
-                                                                'int',
-                                                                u_vec_ref, infinite_time)
-    padded_points_ext = get_padded_points_by_kite_and_tip(variables_xd, n_k, d, periods_tracked, kite, parent,
-                                                                'ext',
-                                                                u_vec_ref, infinite_time)
+    else:
 
-    ring_filaments = get_vortex_ring_filaments(padded_strengths, padded_points_ext, padded_points_int, rdx)
+        fix_shape = (1,1)
+        try:
+            fix_shape = fix.shape
+        except:
+            message = 'An attempt to append bounds was passed a vortex-related constraint with an unaccepted structure.'
+            awelogger.logger.error(message)
 
-    return ring_filaments
+        g_bounds['ub'].append(cas.DM.zeros(fix_shape))
+        g_bounds['lb'].append(cas.DM.zeros(fix_shape))
 
+        return g_bounds
 
+def get_vortex_verification_mu_vals():
 
+    radius = 155.77
```

Comment:
moved into awebox.viz.wake so that it is with the other haas_test specifics. will try to figure out how to do this without needing "magic numbers"...

Reply:
fixed.